orthoinference/README.md

```diff
@@ -2,29 +2,33 @@
+
+After all valid ReactionlikeEvents instances have been inferred for a species, the final step is to populate the <b>Pathway</b> instances these RlEs are found in. Pathway inference takes place in its entirety in the <a href="https://github.com/reactome/data-release-pipeline/blob/develop/orthoinference/src/main/java/org/reactome/orthoinference/HumanEventsUpdater.java">HumanEventsUpdater</a> class, and involves creating the hierarchy structure from the Human Pathway equivalent (this will be further expanded at a future date).
 
-After all valid ReactionlikeEvents instances have been inferred for a species, the final step is to populate the <b>Pathway</b> instances these RlEs are found in. Pathway inference takes place in its entirety in the <a href="https://github.com/reactome/data-release-pipeline/blob/develop/orthoinference/src/main/java/org/reactome/orthoinference/HumanEventsUpdater.java">HumanEventsUpdater</a> class, and involves creating the hierarchy structure from the Human Pathway equivalent (this will be further expanded at a future date). 
+Once the database has been updated with inferred Pathways, orthologous PathwayDiagram generation takes place. This generates the diagrams that are visible in Reactome's Pathway Browser for each species. These diagrams are based off the existing Human diagrams in the database.
 
 <h3> Preparing Orthoinference </h3>
 
 Orthoinference can be run once the <a href="https://github.com/reactome/data-release-pipeline/tree/feature/update-stable-ids">UpdateStableIds</a> step has been run. Historically, it had been run following the now-deprecated MyISAM step. Before running the new Java Orthoinference code, there are a few requirements:<br>
 
-- Orthopair file generation must have been completed
-- <a href="https://github.com/reactome/Release/blob/master/scripts/release/orthoinference/tweak_datamodel.pl">tweak_datamodel.pl</a> has been deprecated. This step produced the `test_reactome_##` database with some data model modifications. These modifications aren't required anymore, but the user now needs to create `release_current/test_reactome_##` from `slice_current/test_slice_##`.
+- <a href="https://github.com/reactome/data-release-pipeline/tree/develop/orthopairs">Orthopair</a> file generation must have been completed.
+- The `slice_current` database will need to be dumped and restored to a new `release_current` database.
 - Set the `config.properties` file
-- Locally install a build of <a href="https://github.com/reactome/data-release-pipeline/tree/develop/release-common-lib">release-common-lib</a>, following the instructions at the link
-- `normal_event_skip_list.txt` needs to be placed in `src/main/resources/` folder
+- Locally install a build of <a href="https://github.com/reactome/data-release-pipeline/tree/develop/release-common-lib">release-common-lib</a>, following the instructions at the link.
+- `normal_event_skip_list.txt` needs to be placed in `src/main/resources/` folder.
```

Comment:
These two would be good to add to the release notes for things we want to automate.  I think the `setup_orthoinference.pl` script does them, but it is implemented in Perl and should probably be ported.